### PR TITLE
Add zh_CN localization of manual .and tried to fix the infinite loop bug

### PR DESCRIPTION
**Pull request by loveyanbei**
Build failed at reobf stage showing me MALFORMED, I have no access to coloredlights repo, maybe it's down?
Hoping a test.

### CODE REVIEW

**Comment by payonel** [comment]
i assume this would close: #2395 ?

**Comment by loveyanbei** [comment]
@payonel maybe ok

**Comment by 3TUSK** [comment]
Is this full translation of the entire OC manual?

File: src/main/scala/li/cil/oc/client/gui/Manual.scala [hunk 1]
```diff
@@ -57,7 +57,7 @@ class Manual extends GuiScreen with traits.Window {
     }
 
   def refreshPage(): Unit = {
-    val content = Option(api.Manual.contentFor(ManualAPI.history.top.path)).
+      val content = Option(api.Manual.contentFor(ManualAPI.history.top.path)).
```
Comment (Vexatos) [inline src/main/scala/li/cil/oc/client/gui/Manual.scala]: This does not belong here.

Comment (loveyanbei) [inline src/main/scala/li/cil/oc/client/gui/Manual.scala]: Nothing changed but some spaces, maybe an occasional accident。

**Comment by FledgeXu** [comment]
sorry,I add some translations to the zh_CN.lang,should i flow this PR ?

**Comment by FledgeXu** [comment]
@loveyanbei Will you keep translating the manaul？If not, I'll translate the manual .

**Comment by 3TUSK** [comment]
Any updates on this?

**Pull request merged**